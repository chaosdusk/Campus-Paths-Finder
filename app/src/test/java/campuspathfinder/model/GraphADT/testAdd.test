CreateGraph graph1
# adds two nodes with an edge in between
AddNode graph1 node1
AddNode graph1 node2
AddEdge graph1 node1 node2 label1
# checks result
ListChildren graph1 node1