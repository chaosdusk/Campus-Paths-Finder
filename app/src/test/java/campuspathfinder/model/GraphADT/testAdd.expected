created graph graph1
# adds two nodes with an edge in between
added node node1 to graph1
added node node2 to graph1
added edge label1 from node1 to node2 in graph1
# checks result
the children of node1 in graph1 are: node2(label1)